src/ifcopenshell-python/ifcopenshell/ids.py

```diff
@@ -270,8 +263,7 @@ def asdict(self):
         # if older python collections.OrderedDict()
         spec_dict = {
             "@name": self.name,
-            "@use": self.use,
-            "@ifcVersion": self.ifcVersion,
+            "@necessity": self.necessity,
```

Comment:
I don't see this in either the IDS v4 or v5 XSDs?

Reply:
'necessity' was in XSD for some time, but got replaced by 'use'. Now it will probably also be replaced by minOccur in IDS XSD.